assets/js/state/softwareUpdates.js

```diff
@@ -0,0 +1,44 @@
+import { createAction, createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  loading: false,
+  softwareUpdates: [],
+  errors: [],
+};
+
+export const softwareUpdatesSlice = createSlice({
+  name: 'softwareUpdates',
+  initialState,
+  reducers: {
+    startLoadingSoftwareUpdates: (state) => {
+      state.loading = true;
+    },
+    setSoftwareUpdates: (state, { payload: { hostId, softwareUpdates } }) => {
+      state.loading = false;
+      state.softwareUpdates = state.softwareUpdates
+        .filter((su) => su.hostId !== hostId)
+        .concat({ hostId, softwareUpdates });
+    },
+    setEmptySoftwareUpdates: (state) => {
```

Comment:
related to the comment in the saga: in this way we are clearing up the whole state, regardless of the host.

We either keep in this state the available software updates for one host at a time or if we intend to keep the available software updates for all the hosts, we might want to have `setEmptySoftwareUpdates` empty only the host relevant data.